src/onelogin/saml2/utils.py

```diff
@@ -897,10 +909,8 @@ def validate_node_sign(signature_node, elem, cert=None, fingerprint=None, finger
             dsig_ctx.key = xmlsec.Key.from_memory(cert, xmlsec.KeyFormat.CERT_PEM, None)
 
         dsig_ctx.set_enabled_key_data([xmlsec.KeyData.X509])
-        try:
-            dsig_ctx.verify(signature_node)
-        except Exception:
-            raise Exception('Signature validation failed. SAML Response rejected')
+        dsig_ctx.verify(signature_node)
```

Comment:
I wanted to offer a direct way to get the real cause when a signature validation fails.
instead of catch the exception here on the validate_node_sign method, but I think I can provide this info on an extra parameter. Let me fix that